source/ed/tests/fusioparser_test.cpp

```diff
@@ -48,6 +48,7 @@ struct logger_initialized {
 BOOST_GLOBAL_FIXTURE( logger_initialized )
 
 const std::string ntfs_path = std::string(navitia::config::fixtures_dir) + "/ed/ntfs";
+const std::string sync_ntfs_path = std::string(navitia::config::fixtures_dir) + "/ed/sync_ntfs";
```

Comment:
It's just that the name of the dir is not very clear.

It's a fixture with the new version of ntfs ? can we call it /ed/ntfs_v{N} ?